kong/runloop/certificate.lua

```diff
@@ -302,7 +308,7 @@ end
 
 local function fetch_ca_certificates(ca_ids)
   local cas = new_tab(#ca_ids, 0)
-  local key = new_tab(1, 0)
+  local key = new_tab(0, 1)
```

Comment:
Here key was used by form `key.id`, I think it is a hash table, so we should use `new_tab(0, 1)`.

Reply:
why even create a table when there is only 1?

Reply:
You could have module level:
```
local CA_KEY = {
  id = ""
}
```

and then just use:
```
CA_KEY.id = ca_id
local obj, err = kong.db.ca_certificates:select(CA_KEY)
```

Reply:
good idea, thank you @bungle